perses/app/fah_generator.py

```diff
@@ -0,0 +1,203 @@
+#1/usr/bin/env python
+
+__author__ = 'dominic rufa'
+
+"""
+Folding@Home perses executor
+
+# prereqs
+conda config --add channels omnia --add channels conda-forge
+conda create -n perses python3.7 perses tqdm dicttoxml
+pip uninstall --yes openmmtools
+pip install git+https://github.com/choderalab/openmmtools.git
+
+argv[1]: setup.yaml (argument for perses.app.setup_relative_calculation.getSetupOptions)
+argv[2]: neq_setup.yaml (contains keywords for openmmtools.integrators.PeriodicNonequilibriumIntegrator arguments)
+argv[3]: run_number (project run number; defined by f"setup_options['trajectory_directory']_phase/RUN_{run_number}")
+"""
+import yaml
```

Comment:
As a general practice, let's always keep `import`s local to right before they're used.
This is generally useful as it keeps the code tidier and more modular, and prevents long `import` times.